tests/utils/sc_linac/test_cavity.py

```diff
@@ -345,6 +345,11 @@ def test_edm_macro_string_rack_b(cavity):
     assert cav.edm_macro_string == "C=5,RFS=1B,R=B,CM=ACCL:L0B:01,ID=01,CH=1"
 
 
+def test_cryo_edm_macro_string(cavity):
+    cavity = MACHINE.cryomodules["01"].cavities[1]
+    assert cavity.cryo_edm_macro_string == "CM=01,AREA=L0B"
```

Comment:
So you can delete the previous line and just have something like `assert cavity.cryo_edm_macro_string == f"CM={cavity.cryomodule.name},AREA={cavity.linac.name}"`